### ISSUE TITLE
Add browser support policy

### ISSUE THREAD

**Issue by locks** [emberjs/website#2870]
See https://github.com/emberjs/ember.js/issues/10566 for more context.

**Comment by mgol** [comment]
Any updates?

**Comment by locks** [comment]
@mgol nope, haven't gotten round to it and neither has someone else. We support IE9+, and current versions - 1 of the evergreen browsers, as mentioned in the linked issue :)

**Comment by mgol** [comment]
What about the Android Browser? Is it supported at all?

**Comment by locks** [comment]
@mgol do you literally mean Android Browser, not Chrome?

**Comment by mgol** [comment]
Yes.

**Comment by mgol** [comment]
Any updates here? Is this still planned?

**Comment by WebStew** [comment]
Hi All,

I am struggling to find this information as well. This sort of information is extremely important when trying to put forward a case to the business to use Ember. 

In my research of all the current popular frameworks Ember has been rock solid in all aspects once you get through some of the hype of the others - I was a little surprised not to find this information within seconds.

Hope this comes soon and keep up the great work!

**Comment by rwjblue** [comment]
@WebStew - I definitely agree with your sentiment. We (as a team) have gone back and forth quite a bit on this topic over the years, let me add it to the agenda for our next meeting to see if we can get this resolved...

**Comment by mgol-sumo** [comment]
@rwjblue has this been discussed?

**Comment by mgol** [comment]
@rwjblue has this been discussed?

**Comment by zsoldosp** [comment]
this issue and #3139 seem to be duplicates, and #3139 has more recent comments (but still no definitive answer :( )
